docs/docs/tutorials/noirjs_app.md

```diff
@@ -1,122 +1,101 @@
 ---
-title: Tiny NoirJS app
-description: Learn how to setup a new app that uses Noir to generate and verify zero-knowledge SNARK proofs in a typescript or javascript environment
-keywords: [how to, guide, javascript, typescript, noir, barretenberg, zero-knowledge, proofs]
+title: Building a web app with NoirJS
+description: Learn how to setup a new app that uses Noir to generate and verify zero-knowledge SNARK proofs in a typescript or javascript environment.
+keywords: [how to, guide, javascript, typescript, noir, barretenberg, zero-knowledge, proofs, app]
 sidebar_position: 0
 ---
 
-NoirJS works both on the browser and on the server, and works for both ESM and CJS module systems. In this page, we will learn how can we write a simple test and a simple web app to verify the standard Noir example.
+NoirJS is a set of packages meant to work both in a browser and a server environment. In this tutorial, we will build a simple web app using them. From here, you should get an idea on how to proceed with your own Noir projects!
 
 You can find the complete app code for this guide [here](https://github.com/noir-lang/tiny-noirjs-app).
```

Comment:
Looks like the code here needs to updated as well.

Reply:
Yep it's ready but I want to push it only when this hits stable...

Reply:
Isn't 0.19 stable? the tiny-noirjs-app repo is using 0.17

Reply:
Sorry I mean when this tutorial hits the stable version of the docs 😂 otherwise people would look at the old tutorial which wouldn't match with the version on the repo